RealmTasks Android/app/src/main/java/io/realm/realmtasks/list/TaskAdapter.java

```diff
@@ -107,7 +107,9 @@ public void onItemDismissed(final int position) {
         realm.executeTransaction(new Realm.Transaction() {
             @Override
             public void execute(Realm realm) {
+                final Task task = getData().get(position);
                 getData().remove(position);
```

Comment:
Do we need this? I think `task.deleteFromRealm();` removes it from the RealmList